suite-native/module-passphrase/src/components/PassphraseScreenHeader.tsx

```diff
@@ -44,8 +75,23 @@ export const PassphraseScreenHeader = () => {
                 colorScheme="tertiaryElevation1"
                 accessibilityRole="button"
                 accessibilityLabel="close"
-                onPress={handleClose}
+                onPress={handlePress}
             />
+            <BottomSheet isVisible={shouldShowWarningBottomSheet} onClose={handleCloseBottomSheet}>
+                <VStack justifyContent="center" alignItems="center" padding="small" spacing="large">
+                    <Text variant="titleSmall" textAlign="center">
+                        <Translation id="modulePassphrase.confirmOnDevice.warningSheet.title" />
+                    </Text>
+                    <VStack style={applyStyle(buttonWrapperStyle)}>
+                        <Button colorScheme="redBold" onPress={handleClose}>
```

Comment:
Since there will be some changes here and there, I already created a task for this and will discuss with QA when this whole EPIC will be nearly finished where they would need those testIDs and add them all together.
https://github.com/trezor/trezor-suite/issues/12176